### PR TITLE
IDE-Cleanup

### PR DESCRIPTION
**Pull request by RoiEXLab**
This is another PR that runs IDE-automated-cleanup.

### CODE REVIEW

File: game-core/src/main/java/tools/map/making/MapPropertiesMaker.java [hunk 1]
```diff
@@ -323,14 +323,6 @@ private void loadProperties() {
...
         ToolLogger.error("Failed to load map properties: " + centerName, e);
       }
-      for (final Method setter : mapProperties.getClass().getMethods()) {
-        final boolean startsWithSet = setter.getName().startsWith("set");
-        if (!startsWithSet) {
-          continue;
-        }
-
-        // TODO: finish this
-      }
```
Comment (RoiEXLab) [inline game-core/src/main/java/tools/map/making/MapPropertiesMaker.java]: I removed the entire loop because it didn't do anything.

**Comment by RoiEXLab** [comment]
I reviewed this PR twice now.
I'm going to self merge it do be able to open another PR with more simple cleanups.
After that more complex cleanups will follow.
If there are any objections regarding a formatting choice or similar I'd be happy to adress this in a follow-up PR.

File: game-core/src/main/java/games/strategy/triplea/ui/EconomyPanel.java [hunk 2]
```diff
@@ -60,7 +60,7 @@ private void setResourceColumns() {
         }
         statList.add(new ResourceStat(resource));
       }
-      statsResource = statList.toArray(new IStat[statList.size()]);
+      statsResource = statList.toArray(new IStat[0]);
```
Comment (ssoloff) [inline game-core/src/main/java/games/strategy/triplea/ui/EconomyPanel.java]: FWIW, here's the original [reference](https://shipilev.net/blog/2016/arrays-wisdom-ancients/) that provides the justification for the IDE static analysis message in 4ed75fd96.

**Comment by DanVanAtta** [comment]
@RoiEXLab "If there are any objections regarding a formatting choice or similar I'd be happy to adress this in a follow-up PR."

current project convention is that we follow up with formatting when doing the auto-refactor. I opened an issue to schedule an agreed time to do large refactors. @ron-murhammer was concerned about potential conflicts such large scale changes could cause to in-flight work.

**Pull request merged**